### PR TITLE
Parallel coord

### PR DESCRIPTION
**Pull request by doutriaux1**
goes with https://github.com/UV-CDAT/uvcdat/pull/2130

### CODE REVIEW

File: vcs/Canvas.py [hunk 1]
```diff
@@ -2255,7 +2255,8 @@ def drawtextcombined(self, Tt_name=None, To_name=None, string=None,
                 *******************Texttable Names List**********************
                 ...
                 *******************End Texttable Names List**********************
-                >>> tc=a.drawtextcombined(Tt_name = 'std_example', To_name='7left_example', string='Hello example!', spacing=5,
+                >>> tc=a.drawtextcombined(Tt_name = 'std_example', To_name='7left_example',
+                ...                   string='Hello example!', spacing=5,
```
Comment (doutriaux1) [inline vcs/Canvas.py]: @embrown do you think my split will break anything for you?

File: vcs/template.py [hunk 2]
```diff
@@ -1465,28 +1465,70 @@ def scalefont(self, scale):
...
-        This plots the template stuff on the Canvas.
-        It needs a slab and a graphic method.
+    def drawLinesAndMarkersLegend(self, canvas,
+                                  linecolors, linetypes, linewidths,
+                                  markercolors, markertypes, markersizes,
+                                  strings, bg=False, render=True):
+        """Draws a legend with line/marker/text inside a template legend box
+    Auto adjust text size to make it fit inside the box
+    Auto arrange the elements to fill the box nicely
+    :: Example ::
```
Comment (embrown) [inline vcs/template.py]: @doutriaux1 format is incorrect here. Should be:
:Example:
    .. doctest:: template_drawLinesAndMarkers
        >>> ...
Where the ... is your code example. Any python lines that would return something to the screen when run on the console should be followed by the expected output:
>>> print("Like this")
'Like this'

File: vcs/Canvas.py (hunk 1 above)
Comment (embrown) [inline vcs/Canvas.py]: @doutriaux1 No, this should be fine.

File: vcs/template.py [hunk 3]
```diff
@@ -1465,28 +1465,70 @@ def scalefont(self, scale):
...
-        """
+    import vcs
+    x = vcs.init()
+    t = vcs.createtemplate()
+    t.drawLinesAndMarkersLegend(x,
+          ["red","blue","green"], ["solid","dash","dot"],[1,4,8],
+          ["blue","green","red"], ["cross","square","dot"],[3,4,5],
+          ["sample A","type B","thing C"],True)
+    x.png("sample")
+
```
Comment (embrown) [inline vcs/template.py]: @doutriaux1 format is incorrect here. Should be:
```
:Example:
    
    .. doctest:: template_drawLinesAndMarkers
        
        >>> ...
```
Where the ... is your code example. Any python lines that would return something to the screen when run on the console should be followed by the expected output:
```
>>> print("Like this")
'Like this'
```

**Pull request merged**